exhibitor-standalone/src/main/java/com/netflix/exhibitor/standalone/ExhibitorCreator.java

```diff
@@ -230,7 +233,19 @@ else if ( "true".equalsIgnoreCase(commandLine.getOptionValue(FILESYSTEMBACKUP))
             servoRegistration = new ServoRegistration(new JmxMonitorRegistry("exhibitor"), 60000);
         }
 
-        String              preferencesPath = commandLine.getOptionValue(PREFERENCES_PATH);
+        HttpsConfiguration httpsConfiguration = HttpsConfiguration.builder()
+            .serverKeystorePath(env.get(ExhibitorEnv.SERVER_KEYSTORE_PATH))
+            .serverKeystorePassword(env.get(ExhibitorEnv.SERVER_KEYSTORE_PASSWORD))
+            .clientKeystorePath(env.get(ExhibitorEnv.CLIENT_KEYSTORE_PATH))
+            .clientKeystorePassword(env.get(ExhibitorEnv.CLIENT_KEYSTORE_PASSWORD))
+            .truststorePath(env.get(ExhibitorEnv.TRUSTSTORE_PATH))
+            .truststorePassword(env.get(ExhibitorEnv.TRUSTSTORE_PASSWORD))
+            .requireClientCert(env.get(ExhibitorEnv.REQUIRE_CLIENT_CERT))
+            .verifyPeerCert(env.get(ExhibitorEnv.VERIFY_PEER_CERT))
+            .build();
```

Comment:
The `build` method checks if the provided configuration is valid. If nothing is passed at all the class will be build but `EXHIBITOR_TLS_SERVER_KEYSTORE_PATH' will be `null` and TLS will not be used. There is also a unit test for this case.